assets/training/model_management/components/import_model/spec.yaml

```diff
@@ -4,7 +4,7 @@ type: pipeline
 name: import_model
 display_name: Import model
 description: Import a model into a workspace or a registry
-version: 0.0.21
+version: 0.0.22
```

Comment:
lets perform version update in a sep. PR. 

Here is the sample PR you can take a look at: https://github.com/Azure/azureml-assets/pull/2017

Reply:
Done